src/styles/styles.css

```diff
@@ -70,9 +75,4 @@ div.row2 .modalBtn:visited {
 
 .ant-modal-close-x {
     color: rgb(255, 255, 255, 0.85);
-}
-
-
-
-
-
+}
```

Comment:
It looks like there are merge conflicts in this file. Just pull/push changes and resolve the merge conflicts and we're good to go.

Reply:
Got with you on this, should be resolved!

Reply:
Resolved it.